src/exporter/components/export/export-dialog.tsx

```diff
@@ -138,10 +142,14 @@ export const ExportDialog: React.FC<ExportDialogProps> = observer((props) => {
 
   useEffect(() => {
     if (exporterStore.hasError(ExportStoreError.DUPLICATE_PATH)) {
-      setServerErrors({ duplicate: 'export.dialog.duplicate-path.text' });
+      setServerErrors({ ...serverErrors, duplicate: 'export.dialog.duplicate-path.text' });
       exporterStore.cleanError(ExportStoreError.DUPLICATE_PATH);
     }
-  }, [exporterStore, exporterStore.errors]);
+    else if(exporterStore.hasError(ExportStoreError.BBOX_TOO_SMALL_FOR_RESOLUTION)) {
```

Comment:
is there a test for multiple errors? 
for the "setServerErrors" and clean specific error while other remains?

Reply:
Added test